core/vector.h

```diff
@@ -31,11 +31,12 @@
 #ifndef VECTOR_H
 #define VECTOR_H
 
-/**
+/*
```

Comment:
Ok, I can revert these preference based comments similar to this one you mention. And I won't try and implement a commenting system until a consensus is reached.
I'll just try and change them for readability and consistency.

Reply:
Yeah some contributors have IDEs that use a different style (the `/* FILENAME_H_ */` style is maybe from @hpvb's Eclipse?). But the majority are of the form `// FILENAME_H`, so that's what I try to enforce little by little. If that can be configured somehow in `.editorconfig`, that would be nice.

Reply:
Oops, wrong line.